Objective-C/Internal/Replicator/CBLTrustCheck.mm

```diff
@@ -132,7 +145,21 @@ - (NSURLCredential*) checkTrust: (NSError**)outError {
     
     // Evaluate trust:
     SecTrustResultType result;
-    OSStatus err = SecTrustEvaluate(_trust, &result);
+    OSStatus err;
+#if TARGET_OS_MACCATALYST
+    BOOL trusted = NO;
+    CFErrorRef cfError = nullptr;
+    if (@available(iOS 12.0, macos 10.14, *)) {
+        trusted = SecTrustEvaluateWithError(_trust, &cfError);
```

Comment:
If we don't care about the result here, can we don't get the trusted boolean value and use NULL for the error?

Reply:
ok

Reply:
Return value('trusted') cannot be ignored, it will generate error . So adding a verbose log specifying the failure and evaluating the trust result. 
removed the cferror.